pkg/cluster/meta/topology.go

```diff
@@ -226,7 +226,11 @@ func checkStoreStatus(storeAddr string, pdList ...string) string {
 		}
 	}
 	if latestStore != nil {
-		return latestStore.Store.StateName
+		state := latestStore.Store.StateName
+		if strings.ToLower(state) == "offline" {
+			state = "Pending Offline" // avoid misleading
```

Comment:
I don't think this is the proper way. I think we add `Pending`  suffix just when both `spec.Offline` and `status==offline` are true. It's better to change the status display string at  https://github.com/pingcap/tiup/pull/447/files#diff-dc4c66f2c87a0397b0ad841b79378f17R217